docs/konflux/release_operator_with_konflux.md

```diff
@@ -50,26 +50,26 @@ oc get snapshots --sort-by .metadata.creationTimestamp -l pac.test.appstudio.ope
 
 Example:
 ```console
-helm-operator-v2wsd		False	Merge pull request #170 from parodos-dev/appstudio-orchestrator-operator-bundle
+helm-operator-v2wsd		False	Merge pull request #170 from rhdhorchestrator/appstudio-orchestrator-operator-bundle
```

Comment:
@jenniferubah @jordigilh is this OK?

Reply:
Yes this looks fine to me. Will need to update our git repository in our Konflux configuration. Note to myself.

Reply:
Actually, I can see a PR for those changes. Jordi is already working on it.